src/lib/utils/wagmi.ts

```diff
@@ -1,72 +1,54 @@
-import { connectorsForWallets } from '@rainbow-me/rainbowkit'
-import {
-  argentWallet,
-  braveWallet,
-  coinbaseWallet,
-  ledgerWallet,
-  metaMaskWallet,
-  okxWallet,
-  rainbowWallet,
-  safeWallet,
-  trustWallet,
-  walletConnectWallet,
-} from '@rainbow-me/rainbowkit/wallets'
-import { Chain, http } from 'viem'
-import { createConfig } from 'wagmi'
+import { defaultWagmiConfig } from '@web3modal/wagmi/react/config'
+import { cookieStorage, createStorage, http } from 'wagmi'
 import { arbitrum, base, localhost, mainnet } from 'wagmi/chains'
 
-import { AlchemyApiKey } from '../../constants/server'
+export const projectId = process.env.NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID
+
+if (!projectId) {
+  throw new Error('Project ID is not defined')
+}
 
 const isDevelopmentEnv = process.env.NEXT_PUBLIC_VERCEL_ENV === 'development'
 const isPreviewEnv = process.env.NEXT_PUBLIC_VERCEL_ENV === 'preview'
 const shouldShowLocalHost = isDevelopmentEnv || isPreviewEnv
 
-const projectId = process.env.NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID!
-
-const connectors = connectorsForWallets(
-  [
-    {
-      groupName: 'Recommended',
-      wallets: [
-        safeWallet,
-        metaMaskWallet,
-        rainbowWallet,
-        argentWallet,
-        coinbaseWallet,
-        ledgerWallet,
-        okxWallet,
-      ],
-    },
-    {
-      groupName: 'Others',
-      wallets: [walletConnectWallet, braveWallet, trustWallet],
-    },
-  ],
-  {
-    appName: 'Index App',
-    projectId,
-  },
-)
+// Create a metadata object
+export const metadata = {
+  name: 'indexcoop-app',
+  description: 'IndexCoop App',
+  url: 'https://app.indexcoop.com',
+  icons: ['/index-logo-black.png'],
+}
 
-const lh = { ...localhost, id: 31337 }
-
-export const chains: [Chain, ...Chain[]] = [
-  mainnet,
+export const chains = [
   arbitrum,
+  mainnet,
   base,
-  ...(shouldShowLocalHost ? [lh] : []),
-]
+  ...(shouldShowLocalHost ? [localhost] : []),
+] as const
 
-export const wagmiConfig = createConfig({
+export const config = defaultWagmiConfig({
   chains,
-  connectors,
+  projectId,
+  metadata,
   ssr: true,
+  auth: {
+    email: false,
+    socials: [],
+  },
+  storage: createStorage({
+    storage: cookieStorage,
+  }),
+
   transports: {
+    [mainnet.id]: http(
+      `https://eth-mainnet.g.alchemy.com/v2/${process.env.NEXT_PUBLIC_ALCHEMY_ID}`,
+    ),
     [arbitrum.id]: http(
-      `https://arb-mainnet.g.alchemy.com/v2/${AlchemyApiKey}`,
+      `https://arb-mainnet.g.alchemy.com/v2/${process.env.NEXT_PUBLIC_ALCHEMY_ID}`,
+    ),
+    [base.id]: http(
+      `https://base-mainnet.g.alchemy.com/v2/${process.env.NEXT_PUBLIC_ALCHEMY_ID}`,
     ),
-    [base.id]: http(`https://base-mainnet.g.alchemy.com/v2/${AlchemyApiKey}`),
-    [mainnet.id]: http(`https://eth-mainnet.g.alchemy.com/v2/${AlchemyApiKey}`),
-    [lh.id]: http('http://127.0.0.1:8545/'),
```

Comment:
Localhost is supported automatically with this url?

Reply:
No, i basically did the same thing in this file